queries/roc/textobjects.scm

```diff
@@ -0,0 +1,35 @@
+(anon_fun_expr
+  (expr_body) @function.inner
+) @function.outer
+
+(argument_patterns
+  ((_) @parameter.inner . ","? @parameter.outer) @parameter.outer
+)
+
+(function_type
+  ((_) @parameter.inner . ","? @parameter.outer) @parameter.outer(#not-eq? @parameter.inner "->")
+)
+
+(function_call_expr
+  .
+  (_)
+  (parenthesized_expr (expr_body) @parameter.inner) @parameter.outer
+)
+
+(function_call_expr
+  .
+  (_) ((_) @parameter.inner) @parameter.outer
```

Comment:
These parentheses are also redundant; if this was really meant to do what it is doing (apply inner and outer to the same thing) you can just do `(_) @parameter.inner @parameter.outer`